manifests/dragonfly-operator.yaml

```diff
@@ -47,6 +47,839 @@ spec:
           spec:
             description: DragonflySpec defines the desired state of Dragonfly
             properties:
+              affinity:
```

Comment:
Can you remove the changes in this file? This is updated part of the release process. Updating it now, will cause problems to users who try to follow the README.

Reply:
Yes, I have removed the changes from the commit.